mailpoet/lib/Newsletter/Sending/ScheduledTaskQueuedSubscribersListingRepository.php

```
@@ -0,0 +1,81 @@
+    // The queue table is, by definition, the unprocessed set — there is no
+    // sub-group to filter on.
+  }
+
+  protected function applySorting(QueryBuilder $queryBuilder, string $sortBy, string $sortOrder) {
+    // Ordering by subscriberId is mapped to email for consistency with the
+    // Subscriber listing; the queue exposes no other sortable column.
+    $sortBy = $sortBy === 'subscriberId' ? 's.email' : 'stsq.subscriber';
+    $queryBuilder->addOrderBy($sortBy, $sortOrder);
+  }
+
+  protected function applySearch(QueryBuilder $queryBuilder, string $search, array $parameters = []) {
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$parameters'. (undefined)

(UnusedFormalParameter)